src/components/pacbio/PacbioRunTubeItem.vue

```diff
@@ -10,7 +10,7 @@
       <div class="flex flex-col h-full w-full">
         <div class="flex flex-row w-full p-2 items-center border-b-2">
           <p class="text-xl font-bold w-full px-2 text-left" data-attribute="barcode">
-            {{ barcode }}
+            {{ barcode }} - {{ type }}
```

Comment:
Is the type needed or was it for debugging purposes?

Reply:
The type was there already when I started working. May be, added by Steve. I hope it can be removed.

Reply:
I added this on purpose and it was not for debugging. It means that the type appears on the Pools & Libraries component so you know if it is a pool or library so is a useful indicator. It looks like it has been removed. Can we put it back please?

Reply:
Sorry about that.I will put that back.
Do you agree with this way presenting that information?
![Screenshot 2024-04-10 at 10 00 36](https://github.com/sanger/traction-ui/assets/55585488/8f752166-7ac2-48f4-b25b-dcd6c6d8e4c0)

Reply:
Looks good to me. Better than what I had done.